### ISSUE TITLE
EKS cluster fails to come up in an air gapped proxy setup

### ISSUE THREAD

**Issue by sowmyav27** [rancher/rancher#29070]
**What kind of request is this (question/bug/enhancement/feature request):** bug

**Steps to reproduce (least amount of steps as possible):**

- Deploy an air gapped proxy setup
- Deploy EKS cluster
- EKS cluster fails to come up with error

```
time="2020-09-22T01:22:06Z" level=error msg="error syncing 'cattle-global-data/c-b9x2f': handler eks-controller: error creating stack with VPC template: error creating master: RequestError: send request failed\ncaused by: Post https://cloudformation.us-west-1.amazonaws.com/: dial tcp 176.32.118.28:443: i/o timeout, requeuing"
time="2020-09-22T01:22:06Z" level=info msg="Bringing up vpc"
time="2020-09-22T01:22:44Z" level=error msg="error syncing 'cattle-global-data/c-k9454': handler eks-controller: error creating stack with service role template: error creating master: RequestError: send request failed\ncaused by: Post https://cloudformation.us-west-2.amazonaws.com/: dial tcp 54.240.251.124:443: i/o timeout, requeuing"
time="2020-09-22T01:22:44Z" level=info msg="Creating service role"
```

**Expected Result:**
EKS cluster should come up successfully.

**Other details that may be helpful:**

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): master-head - commit id: `ed2749841db0`
- Installation option (single install/HA): HA

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): hosted EKS
- Kubernetes version (use `kubectl version`):

```
1.17
```

**Comment by paynejacob** [comment]
I traced this down to the container running the eks operator it's self.  I am able to resolve dns inside the container but it cannot route to the aws api server.  I am able to route to the aws api server from the node and k3s container.

**Comment by sowmyav27** [comment]
reopening the issue for a fix in 2.5-head

**Comment by sowmyav27** [comment]
**On 2.5-head commit id: `b722847200` and on master-head - commit id: `4916c38e4`**

- Deploy an air gapped proxy setup
- Deploy EKS cluster
- EKS cluster comes up Active. No error seen
